src/main/java/com/autotune/database/dao/ExperimentDAOImpl.java

```diff
@@ -732,6 +732,29 @@ public List<KruizeExperimentEntry> loadAllExperiments() throws Exception {
         return entries;
     }
 
+    @Override
+    public List<KruizeLMExperimentEntry> loadAllLMExperiments() throws Exception {
+        //todo load only experimentStatus=inprogress , playback may not require completed experiments
+        List<KruizeLMExperimentEntry> entries = null;
+        String statusValue = "failure";
+        Timer.Sample timerLoadAllExp = Timer.start(MetricsConfig.meterRegistry());
```

Comment:
Is this a new timer that is being added? Can you please update this [file](https://github.com/kruize/autotune/blob/master/design/KruizePromQL.md) with the details on what this means